meteor/lib/api/showStyles.ts

```diff
@@ -1,16 +1,25 @@
 import { ShowStyleBaseId } from '../collections/ShowStyleBases'
 import { ShowStyleVariantId } from '../collections/ShowStyleVariants'
+import { IBlueprintConfig } from '@sofie-automation/blueprints-integration'
 
 export interface NewShowStylesAPI {
 	insertShowStyleBase(): Promise<ShowStyleBaseId>
 	insertShowStyleVariant(showStyleBaseId: ShowStyleBaseId): Promise<ShowStyleVariantId>
+	insertShowStyleVariantWithBlueprint(
```

Comment:
What is the different between `insertShowStyleVariant` and `insertShowStyleVariantWithBlueprint`?

Reply:
The difference is that `insertShowStyleVariant` creates an empty new variant, where `insertShowStyleVariantWithBlueprint` creates a variant where everything follows along from the import.

Reply:
Would be nice just to overload the method instead :)

Reply:
To keep the responsibility divided and to avoid too much functionality implemented server side, i will be keeping the two functions but rename the` insertShowStyleVariantWithBlueprint`. Let me know if you want me to do otherwise.